contrib/cirrus/lib.sh

```diff
@@ -290,3 +290,33 @@ remove_packaged_podman_files() {
     # Be super extra sure and careful vs performant and completely safe
     sync && echo 3 > /proc/sys/vm/drop_caches || true
 }
+
+# Execute make localbenchmarks in $CIRRUS_WORKING_DIR/data
+# for preserving as a task artifact.
+localbenchmarks() {
+    local datadir
+    req_env_vars DISTRO_NV PODBIN_NAME PRIV_NAME TEST_ENVIRON TEST_FLAVOR
+    req_env_vars VM_IMAGE_NAME EC2_INST_TYPE
+
+    datadir=$CIRRUS_WORKING_DIR/data
+    mkdir -p $datadir
+
+    (
+      echo "# Metadata reflecting benchmark basis."
+      echo "# Intended for machine consumption."
+
+      printenv
+
+      echo"\
```

Comment:
This would be a great place to add something like
```
# these are not actual environment variables; all of these are included as a representation of system configuration
```

Reply:
Yeah, that's what I was trying to say in the comments.  K.  Let me tweak it some more.